src/v2/Apps/Auction/Components/RegistrationForm.tsx

```diff
@@ -233,7 +249,13 @@ export const RegistrationForm: React.FC<RegistrationFormProps> = props => {
 
         props.onSubmit(actions, result)
       }
-    })
+    } catch (error) {
+      logger.error(error)
+      setSubmitting(false)
+      setStatus(
+        "Something went wrong while processing your bid. Please make sure your internet connection is active and try again"
```

Comment:
Yeah, This error message could be misleading because of a bug (which is basically our fault, not the user's connection or machine). Once we identify common errors we should re-visit and make it more helpful.